sbt/src/sbt-test/source-dependencies/macro-annotation/macros/src/main/scala/Macros.scala

```diff
@@ -3,24 +3,24 @@ import scala.language.experimental.macros
 import scala.annotation.StaticAnnotation
 
 object HelloMacro {
-  def impl(c: Context)(annottees: c.Tree*): c.Tree = {
+  def impl(c: Context)(annotatees: c.Tree*): c.Tree = {
```

Comment:
I get that scala seems to like this invented word. But it isn't right.
Note: as each commit covers a single concept, if you don't like a change, you can trivially drop the commit and move along.